### PR TITLE
Build default cobalt target

### PR DESCRIPTION
**Pull request by niranjanyardi**
b/375045897

### CODE REVIEW

File: cobalt/BUILD.gn [hunk 1]
```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 
 group("gn_all") {
-  # TODO(b/371589344): Fix android build configs.
-  deps = [ "//starboard($starboard_toolchain)" ]
+  if (is_cobalt) {
+    # TODO(b/371589344): Fix android build configs.
```
Comment (andrewsavage1) [inline cobalt/BUILD.gn:17]: I think this TODO can be removed

File: cobalt/BUILD.gn [hunk 2]
```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 
 group("gn_all") {
-  # TODO(b/371589344): Fix android build configs.
-  deps = [ "//starboard($starboard_toolchain)" ]
+  if (is_cobalt) {
```
Comment (andrewsavage1) [inline cobalt/BUILD.gn:16]: If we're adding a condition here, shouldn't it be removed in the toplevel BUILD.gn file?

File: .github/config/linux.json [hunk 3]
```diff
@@ -14,7 +14,7 @@
         "gin_unittests",
         "blink_unittests",
         "content_shell",
-        "starboard/libstarboard.so.17"
+        "cobalt:gn_all"
```
Comment (andrewsavage1) [inline .github/config/linux.json:17]: I think I forgot to add the same target to the android json config, if you're able to do that and it just works that'd be super helpful (or if it fails you can back it out and we can just submit this)

**Review by andrewsavage1 — APPROVED** [review]

File: .github/config/linux.json (hunk 3 above)
Comment (niranjanyardi) [reply .github/config/linux.json:17]: Doing it in a different PR: https://github.com/youtube/cobalt/pull/4302

That PR has a better approach than this one IMO

Comment (niranjanyardi) [reply .github/config/linux.json:17]: I could have added the other one in the same PR but i wasn't sure if builds will fail or what so created a new one.
Next time will try to reuse the same PR .

**Pull request closed without merging**